### PR TITLE
Should solve issue 32.

### PR DESCRIPTION
**Pull request by RexRascerX**
Closes issue 32, hopefully.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,9 +39,14 @@
...
-    "typescript",
+    "issues",
     "github",
-    "gitlab"
+    "gitlab",
+    "projects",
+    "productivity",
+    "tool",
+    "developer",
+    "internet"
```
Comment (robobenklein) [inline package.json]: lol I don't think we need this here

(this is how PR reviews will work, I can leave notes on individual lines and request changes, then when you push a fix to it the comment on the line should collapse and you can re-request a review, try it out with this as the example.)

**Comment by RexRascerX** [comment]
This should close the merge request problem.
This should resolve issue 32 and 35
8dc93de0acc116b6c3fafd60c75b760b5c366dd4

**Comment by RexRascerX** [comment]
I am not authorized to merge this pull request.

**Comment by robobenklein** [comment]
@RexRascerX are you able to merge now?

**Pull request merged**